multiversx_sdk_cli/tests/test_cli_contracts.py

```diff
@@ -39,15 +39,15 @@ def test_contract_new_with_bad_code():
 
 def replace_variable_with_unknown_variable():
     # this is done in order to replace the value added in the adder contract with a unknown variable
-    with open(parent / "testdata-out" / "SANDBOX" / "adder-bad-src" / "src" / "adder.rs", "r") as f:
+    with open(parent / "testdata-out" / "SANDBOX" / "adder-bad-src" / "src" / "adder_bad_src.rs", "r") as f:
```

Comment:
Perhaps specify (in the function name) that it only works with `adder`?

Reply:
Renamed the function to `replace_variable_with_unknown_variable_for_adder()`